src/apps/common/submodules/portWizard/portWizard.scss

```diff
@@ -394,3 +394,15 @@
 #port_wizard_error_dialog {
 	text-align: left;
 }
+
+#navigation_wizard_wrapper .wizard-step {
+	&#step_name_and_numbers {
+		#numbers_to_port_numbers {
+			min-height: 16rem;
+
+			&.monster-invalid + label.monster-invalid {
+				position: static;
+			}
+		}
+	}
+}
```

Comment:
Since you are starting from scratch on this project, maybe you could add a `partials` folder ([ex](https://github.com/2600hz/monster-ui-service-plan-override/tree/master/submodules/itemized)) and use one for each template to separate styles (you can also create a `common` one for styles reused by each step).

Reply:
Sounds good! Can you give me access to `monster-ui-service-plan-override`, so that I can see the example?

Reply:
Unfortunately I can't but it basically looks like this:

```
/moduleName
  /partials
    _common.scss
    _templateNameA.scss
    _templateNameB.scss
    ...
  /views
    _templateNameA.html
    _templateNameB.html
    ...
  moduleName.js
  moduleName.scss
```
And in `moduleName.scss` you simply import your partials nested under the cc root selector.

That way each template can have its own dedicated style sheet.